### ISSUE TITLE
add global timestamp value

### ISSUE THREAD

**Issue by mcenci** [wimdeblauwe/error-handling-spring-boot-starter#68]
As a developer, i would like to add a simple yaml or properties configuration similar to `error.handling.http-status-in-json-response` so that a timestamp would be added to the `ApiErrorResponse` object.

**Comment by wimdeblauwe** [comment]
In what format would you like to have that timestamp? I would propose ISO-8601, but I wonder if the default JVM timezone should be used, or just UTC always?

**Comment by mcenci** [comment]
I would handle it as an ISO-8601 date just like you suggested, UTC is the way, it removes any timezone related pain :-)

**Comment by wimdeblauwe** [comment]
You could create a base exception class and have that field present there, annotated with `@ErrorResponseProperty`. See https://wimdeblauwe.github.io/error-handling-spring-boot-starter/current/#adding-extra-properties-in-the-response

Why exactly do you want to have a timestamp in there? I don't see any practical value in it to be honest.

**Comment by wimdeblauwe** [comment]
I just opened PR #70 which would support your requirement by adding a custom bean like this in your application code:

```
    @Bean
    public ApiErrorResponseCustomizer timestampErrorResponseCustomizer() {
        return new ApiErrorResponseCustomizer() {
            @Override
            public void customize(ApiErrorResponse response) {
                response.addErrorProperty("instant", Instant.now());
            }
        };
    }
```

If you build that branch locally, you can give it a try to see if that is ok for you?

**Comment by wimdeblauwe** [comment]
@mcenci Have you tested this? Does it look ok to you?

**Comment by mcenci** [comment]
@wimdeblauwe it works like a charm!
With some minor spring configuration, thanks to the customizer i can easily handle all cuatom fields i want.

**Comment by wimdeblauwe** [comment]
Implemented in #72

**Comment by wimdeblauwe** [comment]
Version 4.2.0 released with support for this.
